functions/Remove-DbaAgentJob.ps1

```diff
@@ -1,103 +1,105 @@
+            Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
 
-.NOTES
-Author: Sander Stad (@sqlstad, sqlstad.nl)
-Tags: Agent, Job
+        .NOTES
+            Author: Sander Stad (@sqlstad, sqlstad.nl)
+            Tags: Agent, Job
 
-Website: https://dbatools.io
-Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
-License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+            Website: https://dbatools.io
+            Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
+            License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
 
-.LINK
-https://dbatools.io/Remove-DbaAgentJob
+        .LINK
+            https://dbatools.io/Remove-DbaAgentJob
 
-.EXAMPLE
-Remove-DbaAgentJob -SqlInstance sql1 -Job Job1
-Removes the job from the instance with the name Job1
+        .EXAMPLE
+            Remove-DbaAgentJob -SqlInstance sql1 -Job Job1
+        
+            Removes the job from the instance with the name Job1
 
-.EXAMPLE
-Remove-DbaAgentJob -SqlInstance sql1 -Job Job1 -KeepHistory
-Removes the job but keeps the history
+        .EXAMPLE
+            Remove-DbaAgentJob -SqlInstance sql1 -Job Job1 -KeepHistory
+            
+            Removes the job but keeps the history
 
-.EXAMPLE
-Remove-DbaAgentJob -SqlInstance sql1 -Job Job1 -KeepUnusedSchedule
-Removes the job but keeps the unused schedules
+        .EXAMPLE
+            Remove-DbaAgentJob -SqlInstance sql1 -Job Job1 -KeepUnusedSchedule
 
-.EXAMPLE
-Remove-DbaAgentJob -SqlInstance sql1, sql2, sql3 -Job Job1
-Removes the job from multiple servers
+            Removes the job but keeps the unused schedules
 
-.EXAMPLE
-sql1, sql2, sql3 | Remove-DbaAgentJob -Job Job1
-Removes the job from multiple servers using pipe line
+        .EXAMPLE
+            Remove-DbaAgentJob -SqlInstance sql1, sql2, sql3 -Job Job1
+        
+            Removes the job from multiple servers
 
-#>
-    [CmdletBinding(SupportsShouldProcess = $true, ConfirmImpact = "Low")]
+        .EXAMPLE
+            sql1, sql2, sql3 | Remove-DbaAgentJob -Job Job1
+        
+            Removes the job from multiple servers using pipe line
 
+    #>
+    [CmdletBinding(SupportsShouldProcess = $true, ConfirmImpact = "Low")]
     param(
         [parameter(Mandatory = $true, ValueFromPipeline = $true)]
         [Alias("ServerInstance", "SqlServer")]
         [DbaInstanceParameter[]]$SqlInstance,
-
         [Parameter(Mandatory = $false)]
         [PSCredential]$SqlCredential,
-
         [Parameter(Mandatory = $true)]
         [ValidateNotNullOrEmpty()]
         [object[]]$Job,
-
         [Parameter(Mandatory = $false)]
         [switch]$KeepHistory,
-
         [Parameter(Mandatory = $false)]
         [switch]$KeepUnusedSchedule,
-
+        [DbaMode]$Mode = (Get-DbaConfigValue -Name 'message.mode.default' -Fallback "Strict"),
```

Comment:
Curious to know where this is coming from?

Can't recall other commands having this added.